README.md

```diff
@@ -1,24 +1,169 @@
-# README
+# 📖 [Recipe_App] <a name="about-project"></a>
```

Comment:
[OPTIONAL] I suggest changing this name to something more relevant to keep your readme looking professional.(Given that this project is about the "Budget")